travis/upload-to-docker.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 BRANCH=${BRANCH:?'missing BRANCH env var'}
-IMAGE="${REPO:?'missing REPO env var'}:${COMMIT:?'missing COMMIT env var'}"
+IMAGE="${REPO:?'missing REPO env var'}:latest"
```

Comment:
this feels unsafe. Using COMMIT pinpoints the image precisely, while `latest` is kind of vague. What's the harm in keeping COMMIT? Is it because it clutters dockerhub?

Reply:
>  Is it because it clutters dockerhub? 

basically yes, But I don't have anything against it. Although I haven't seen other projects doing it.

Reply:
Would somebody depend on the `SHA` tag? latest features will be present in the `latest` tag and now we have proper release tags.